src/components/Observations/plot/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { useParams } from 'react-router-dom';
+import { APP_PATHS } from 'src/constants';
+import strings from 'src/strings';
+import PageFrame from 'src/components/Observations/common/PageFrame';
+
+export default function ObservationMonitoringPlotDetails(): JSX.Element {
+  const { plantingSiteId, observationId, plantingZoneId, monitoringPlotId } = useParams<{
+    plantingSiteId: string;
+    observationId: string;
+    plantingZoneId: string;
+    monitoringPlotId: string;
+  }>();
+
+  const urlSite = () =>
+    APP_PATHS.OBSERVATION_PLANTING_ZONE_DETAILS.replace(':plantingSiteId', plantingSiteId?.toString());
```

Comment:
Is this a valid URL? It'll still have the `"/:observationId/:plantingZoneId"` segments if I'm reading correctly

Reply:
Good catch - yes, this will only work for the leaf level link. Will fix 👍

Reply:
fixed this, was unable to reuse previously built urls but oh well. correctness is desirable

Reply:
actually, i can simplify this using the hierarchical crumbs..